src/styles/blocks/calendar.scss

```
@@ -0,0 +1,51 @@
+@import "../utils/variables";
+@import "./cells.scss";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: $container-width;
+  padding: $padding-in-container;
+  margin: auto;
+
+  $days:
+    "mon" 0,
+    "tue" $cell-size,
+    "wed" ($cell-size + $gap) * 2,
+    "thu" ($cell-size + $gap) * 3,
+    "fri" ($cell-size + $gap) * 4,
+    "san" ($cell-size + $gap) * 5,
+    "sut" ($cell-size + $gap) * 6;
```

move to the 
```suggestion
  $days:
    "mon" 0,
    "tue" 1,
    "wed" 2,
    "thu" 3,
    "fri"   4,
    "san" 5,
    "sut"  6;
```
Make calculations inside loop

---

Move it to the _variables.scss